<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1171</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Import</string>
    </attribute>
    <widget class="QPushButton" name="pushButtonImportFile">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Import file</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Click on the button bellow to import a file:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Data</string>
    </attribute>
    <widget class="QTableWidget" name="tableData">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>871</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Attributes</string>
    </attribute>
    <widget class="QTableWidget" name="tableAttributes">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>871</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConfirm">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>520</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Anonymize</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Export</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>190</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Click on the button bellow to export the data into file:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonExportData">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>260</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Export data</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
